src/lando/pushlog/management/commands/pushlog_view.py

```diff
@@ -35,19 +44,37 @@ def handle(self, *args, **options):
         if push_id:
             pushes = Push.objects.filter(repo=repo, push_id=push_id)
         else:
-            pushes = Push.objects.filter(repo=repo).order_by("-push_id")
+            pushes = Push.objects.filter(repo=repo)
+
+            if commits_only:
+                if tags_only:
+                    self.stdout.write(
```

Comment:
Seems like these arguments should be in a [mutually exclusive group](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) instead.